Fitting Inspector Issues view into docs (#846)
avohq/docs: pages/workspace/inspector/inspector-events-view.mdx

---
layout: 'guide'
title: 'Inspetor Events View'
abstract: 'How to use the Inspector Events view'
---

import Link from '../../../components/Link';

> The Inspector events view provides a report on all tracking Inspector has seen in your development, staging and production environment. You can use the dashboard to evaluate the health of your tracking implementation and prioritize issues.

Reviewer: ```suggestion
The Inspector events view provides a report on all event tracking Inspector has seen in your development, staging and production environments. You can use the dashboard to evaluate the health of your tracking implementation and prioritize issues.
```

### Filters

Here you can choose which environment to examine, the time window and search for event name.

#### Environments

Currently Inspector supports 3 types of environments:

- Development
- Staging
- Production

The events are streamed to the Environment specified in the client, based on the environment constant provided when initializing Avo Inspector SDK.

> Development and Staging events appear in the dashboard in a few minutes after they are fired and Production events can take up to 2 hours to show up.

#### Time window

Currently Inspector supports 4 types of time windows:

- 1 hour
- 24 hours
- 7 days
- 30 days

> All the time windows are updated hourly on Production and are close to real time in Development and Staging.

Reviewer: This is not true, let's keep it simple for now:
```suggestion
Time windows are updated every few hours on Production and are close to real time in Development and Staging.
```

#### Search for event name

Type the event name in the input field and only events with similar name will be shown.

If you want an exact match put the event name in quotes, e.g. "App Opened".

### Share current view of Inspector

Notice that after you change the header filters the URL also changes. You can share this URL with a colleague and they would see exactly the same slice of inspector data as you filtered.

### Events list

> The main part of Inspector dashboard screen is the list of all events seen. You can expand the events to see the event properties.

Reviewer: ```suggestion
The main part of Inspector events view is the list of all events seen. You can expand the events to see the event properties.
```
